snapshots/terraform/stack/snapshot_scheduler/cloudwatch.tf

```diff
@@ -1,5 +1,7 @@
 resource "aws_cloudwatch_event_rule" "snapshot_scheduler_rule" {
   name                = "snapshot_scheduler_rule-${var.deployment_service_env}"
   description         = "Starts the snapshot_scheduler (${var.deployment_service_env}) lambda"
-  schedule_expression = "rate(1 day)"
+  
+  # At 11pm every day
+  schedule_expression = "cron(23 0 * * ? *)"
```

Comment:
Worth a comment about *why* it's at 11pm each day? You can deduce the comment just from reading the cron.

Reply:
> You can deduce the comment just from reading the cron.
Personally I find cron syntax not easy to parse.